### PR TITLE
feat: add message snackbar component

### PR DESCRIPTION
**Pull request by ChrisBaidoo**
## Description

In this PR we add the MessageSnackbar component which was just in DOMINATE so that it can also be used in MANAGE too. 

## Dependency changes
N/A

## Testing

N/A

## Documentation

N/A

## Migrations (if applicable)

Have any database migrations been committed as part of this pull request?

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] New migrations have been committed
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] If appropriate, I have bumped any version numbers

### CODE REVIEW

**Comment by cooper667** [comment]
Is this different from BaseSnackBar that is already in here?

**Comment by cooper667** [comment]
I think make it a breaking change and remove the BaseSnackBar?

**Comment by ChrisBaidoo** [comment]
> I think make it a breaking change and remove the BaseSnackBar?

How do I make it a breaking change again? By bumping the version number?

File: src/MessageSnackbar.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  const classes = useStyle();
+  return (
+    <ThemeProvider theme={theme}>
+      <Snackbar open={open} onClose={onClose}>
+        <>
+          <SnackbarContent
+            className={classes.snackbarContent}
+            message={
+              <span>
+                <SVG src={imgSrc("warning")} className={classes.svgSmall} />
```
Comment (cooper667) [inline src/MessageSnackbar.tsx:59]: Is this icon ALWAYS going to be warning?

Comment (cooper667) [reply src/MessageSnackbar.tsx:59]: If this component is ONLY for warnings, maybe call it WarningSnackBar? Or make it more generic to maybe take one of our icons?

Comment (ChrisBaidoo) [reply src/MessageSnackbar.tsx]: Changed the named to WarningSnackbar

**Review by cooper667 — APPROVED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**